### PR TITLE
[IMPROVEMENT] Add / Update Docstrings

### LINKED ISSUE #275: [PROPOSAL] Add PEP8 Check In CI
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Sample platform commit (found at the bottom of each page) : **X.X.X**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [ ] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [ ] I have checked that the issue I'm posting isn't already reported.
- [ ] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/sample-platform/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/sample-platform/issues)
- [ ] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never visited/used the platform.
- [ ] I have used the platform just a couple of times.
- [ ] I have used the platform extensively, but have not contributed previously.
- [ ] I am an active contributor to the platform.

---

We should add PEP8 checking in Travis CI in order to enforce a stricter checking on Code style.

https://pypi.org/project/pytest-pep8/

### LINKED ISSUE #276: [BUG] Fix All PEP8 Errors
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Sample platform commit (found at the bottom of each page) : **X.X.X**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [ ] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [ ] I have checked that the issue I'm posting isn't already reported.
- [ ] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/sample-platform/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/sample-platform/issues)
- [ ] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never visited/used the platform.
- [ ] I have used the platform just a couple of times.
- [ ] I have used the platform extensively, but have not contributed previously.
- [ ] I am an active contributor to the platform.

---

Fix all PEP8 issues from the project before implementing #275

### PR DESCRIPTION
**Pull request by thealphadollar**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [x] I am an active contributor to the project.

---

This PR finishes the processing of adding and updating existing docstrings. It brings changes to the following modules:

- mod_home
- mod_regression
- mod_sample
- mod_test
- mod_upload

-- UPDATE --
The PR also fixes all the PEP8 issues. fixes #276 

-- UPDATE 2 --
Added tests for PEP8 issues in Travis. fixes #275

### CODE REVIEW

**Comment by thealphadollar** [comment]
concerns #272

File: mod_home/models_BACKUP_6845.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+"""
```
Comment (canihavesomecoffee) [inline mod_home/models_BACKUP_6845.py]: I don't think you should have added these ;)

File: mod_test/controllers.py [hunk 2]
```diff
@@ -171,12 +171,12 @@ def get_data_for_test(test, title=None):
...
 def get_json_data(test_id):
     """
-    Retrieves the status of a test id and returns it in JSON format.
+    Retrieve the status of a test id and returns it in JSON format.
 
     :param test_id: The id of the test to retrieve data for.
     :type test_id: int
     :return: A JSON structure that holds the data about this test.
-    :rtype: Any
+    :rtype: dict
```
Comment (canihavesomecoffee) [inline mod_test/controllers.py]: Is the return type of `jsonify` dict?

File: mod_home/models_BACKUP_6845.py (hunk 1 above)
Comment (thealphadollar) [reply mod_home/models_BACKUP_6845.py]: Done.

File: mod_test/controllers.py (hunk 2 above)
Comment (thealphadollar) [reply mod_test/controllers.py]: Yes. It's a jsonified dict.

**Pull request merged**